assets/scss/modules/portfolio.scss

```diff
@@ -1,4 +1,56 @@
 .portfolio {
   background-color: $black-haze;
-  min-height: 56.25rem;
+  background: cover;
+  padding-bottom: 7.5rem;
+}
+
+.showcase {
+  padding-top: 5.625rem;
+}
+
+.content {
+  background-color: $picton-blue;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  left: 0;
+  opacity: 0;
+  padding: 0 15px;
+  position: absolute;
+  text-align: center;
+  top: 0;
+  transition: opacity 500ms;
+  width: 100%;
+
+  .description {
+    flex: 1;
+  }
+}
+
+.img-list {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+  .misc-image {
+    list-style: none;
+    margin-left: 5px;
+  }
+
+  .img-list .misc-image {
```

Comment:
Line should be indented 0 spaces, but was indented 2 spaces<br>Merge rule `.img-list .misc-image` with rule on line 31